pkg/integration/tests/status/log_cmd.go

```diff
@@ -0,0 +1,34 @@
+package status
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var LogCmd = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Cycle between two different log commands in the Status view",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig: func(config *config.AppConfig) {
+		config.UserConfig.Git.AllBranchesLogCmd = `echo "view1"`
+		config.UserConfig.Git.AllBranchesLogCmds = []string{`echo "view2"`}
+	},
+	SetupRepo: func(shell *Shell) {},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+
+		t.Views().Status().
+			Focus().
+			Press(keys.Status.AllBranchesLogGraph)
+		t.Views().Main().Content(Contains("view1"))
```

Comment:
I'd maybe also add the check that it _does not contain view2_ at the same time. Just to be extra sure it's not displaying `view1view2` somehow.